### DISCUSSION TITLE
Meetings missing

### DISCUSSION THREAD

**Discussion by shaggione** [code4recovery/12-step-meeting-list#699] [Q&A] [answered]
After Wordpress upgrade, my meetings page is blank. Meetings are listed on the adming page, but when I go to "Where's My Info?
Your public meetings page is [right here](https://kerncountyaa.com/meetings/). Link that page from your site's nav menu to make it visible to the public." there are no meetings shown. How do I connect the page to the data?

**Comment by kiyote33** [comment, marked as answer]
This seems to be a common problem after WordPress upgrades, with a likely solution being to resave the **Settings Permalinks.**  Give that a try...

**Reply by kiyote33** [reply]
The Kern County site meeting list is displaying now, so will mark as answered...
